src/main/kotlin/io/github/techbox/core/modules/commands/CommandRegistry.kt

```diff
@@ -0,0 +1,34 @@
+package io.github.techbox.core.modules.commands
+
+import io.github.techbox.core.modules.ModuleRegistry.ModuleProxy
+import io.github.techbox.utils.logger
+import org.slf4j.Logger
+import java.util.*
+
+
+class CommandRegistry {
+    val log: Logger = logger<CommandRegistry>()
+    private val commands: HashMap<String, ICommand> = HashMap()
+    private val aliases: HashMap<String, String> = HashMap()
```

Comment:
Can you remove this and store the aliases in the command

Reply:
This map is used as a runtime cache to speed up aliases look up on command execution, the aliases are still stored and accessible inside the command.